crates/rpc-types-eth/src/block.rs

```diff
@@ -51,6 +52,32 @@ impl<T: TransactionResponse, H> Block<T, H> {
     }
 }
 
+impl<T> Block<T> {
+    /// Constructs an "uncle block" from the provided header.
+    ///
+    /// This function creates a new [`Block`] structure for uncle blocks (ommer blocks),
+    /// using the provided [`alloy_consensus::Header`].
+    pub fn uncle_block_from_header(header: alloy_consensus::Header) -> Self {
+        Self {
+            uncles: vec![],
+            header: header.clone().seal(header.hash_slow()).into(),
+            transactions: BlockTransactions::Uncle,
+            withdrawals: Some(vec![]),
```

Comment:
this should actually be None I believe

Reply:
With https://github.com/paradigmxyz/reth/pull/12038, sounds ok